connect/transforms/src/test/java/org/apache/kafka/connect/transforms/CastTest.java

```diff
@@ -39,6 +42,7 @@
 public class CastTest {
     private final Cast<SourceRecord> xformKey = new Cast.Key<>();
     private final Cast<SourceRecord> xformValue = new Cast.Value<>();
+    private static final long MILLIS_PER_DAY = 24 * 60 * 60 * 1000;
```

Comment:
minor: consider using `TimeUnit.DAYS.toMillis(1)`

Reply:
Just being consistent with the project's coding conventions: https://github.com/apache/kafka/blob/trunk/connect/api/src/main/java/org/apache/kafka/connect/data/Values.java#L71

Reply:
Could you do a static import for that Values class, then?

Reply:
If a the committer wants, he can. This PR had a ridiculous back and forth and have been stretched for ages compared to what it covers.